index.js

```diff
@@ -49,7 +62,18 @@ getUsersCreditDetails(usersArray);
 // ***************************************************************************
 
 const genderView = users => {
-  // Your code goes here ...
+  const femaleUsers = [];
+  const maleUsers = [];
+  for (let user of users) {
+    const {firstName,lastName,gender} = user;
+    if(gender==='female') {
+      femaleUsers.push(`${firstName} ${lastName}`)
+    } else {
+      maleUsers.push(`${firstName} ${lastName}`)
+    }
+  }
+  const allUsers = {femaleUsers: femaleUsers, maleUsers: maleUsers}
```

Comment:
with ES6, a new **glorious** way of writing object was added - the shorthand.

```js
const allUsers = { femaleUsers, maleUsers }
```

Okay, its not that glorious, but its shorter :)